bigtable/client/testing/inprocess_admin_client.h

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_BIGTABLE_CLIENT_TESTING_INPROCESS_ADMIN_CLIENT_H_
+#define GOOGLE_CLOUD_CPP_BIGTABLE_CLIENT_TESTING_INPROCESS_ADMIN_CLIENT_H_
+
+#include "bigtable/client/admin_client.h"
+#include "bigtable/client/client_options.h"
+#include <google/bigtable/admin/v2/bigtable_table_admin.grpc.pb.h>
+
+namespace btadmin = ::google::bigtable::admin::v2;
+
+namespace bigtable {
+namespace testing {
+
+/**
+ * Connects to Cloud Bigtable's administration APIs.
+ *
+ * This class is mainly for testing purpose, it enable use of a single embedded
+ * server
+ * for multiple test cases. This adminlient uses a pre-defined channel.
+ */
+class InProcessAdminClient : public bigtable::AdminClient {
+ public:
+  InProcessAdminClient(std::string project, ClientOptions options,
+                       std::shared_ptr<grpc::Channel> channel)
+      : project_(std::move(project)),
+        options_(std::move(options)),
```

Comment:
As far as I can see `options_` is never used, why not remove it?  That would let you simplify the test some because you won't need to create a `ClientOptions` object.

Reply:
removed